lib/multiplexer/multiplexer.go

```diff
@@ -501,10 +503,10 @@ func (p Protocol) String() string {
 
 var (
 	proxyPrefix      = []byte{'P', 'R', 'O', 'X', 'Y'}
-	proxyV2Prefix    = []byte{0x0D, 0x0A, 0x0D, 0x0A, 0x00, 0x0D, 0x0A, 0x51, 0x55, 0x49, 0x54, 0x0A}
+	ProxyV2Prefix    = []byte{0x0D, 0x0A, 0x0D, 0x0A, 0x00, 0x0D, 0x0A, 0x51, 0x55, 0x49, 0x54, 0x0A}
 	sshPrefix        = []byte{'S', 'S', 'H'}
 	tlsPrefix        = []byte{0x16}
-	proxyHelloPrefix = []byte(sshutils.ProxyHelloSignature)
```

Comment:
I think that's easier if you move it into https://github.com/gravitational/teleport/blob/ad578958735e865222555822e7ebdc3d362d894d/constants.go
And from there, you replace every reference to ProxyHelloSignature to use `teleport.ProxyHelloSignature`.
This avoids defining the same constant in two different places

Reply:
Right, I actually planned to do that, but then didn't return to it 🤦 
Done.